firestore/vector.go

```diff
@@ -26,25 +26,13 @@ const (
 	valueKey      = "value"
 )
 
-// Vector is an embedding vector.
-type Vector []float64
-
-// ToVector converts float32 or float64 slice to Firestore embedding vector.
-func ToVector[fType float32 | float64](arr []fType) Vector {
-	var arrAny interface{}
-	if arrFloat64, ok := arrAny.([]float64); ok {
-		return Vector(arrFloat64) // Type assertion, no conversion needed
-	}
-
-	vec := make(Vector, len(arr))
-	for i, val := range arr {
-		vec[i] = float64(val)
-	}
-	return vec
-}
+// Vector64 is an embedding vector.
```

Comment:
"is an embedding vector of float64s."

And similarly for Vector32.

Reply:
Updated